crates/core/src/tracing.rs

```diff
@@ -1114,6 +858,28 @@ enum EventKind {
     },
 }
 
+impl EventKind {
+    const CONNECT: u8 = 0;
+    const PUT: u8 = 1;
+    const GET: u8 = 2;
+    const ROUTE: u8 = 3;
+    const SUBSCRIBED: u8 = 4;
+    const IGNORED: u8 = 5;
+    const DISCONNECTED: u8 = 6;
+
+    const fn varint_id(&self) -> u8 {
```

Comment:
Is this still used for anything?

Reply:
Yes, this is used in the header. By using this id, we can improve the `get_router_events`.